application/comit_node/src/http_api/problem.rs

```diff
@@ -50,14 +42,16 @@ pub fn unsupported() -> HttpApiProblem {
     HttpApiProblem::new("swap-not-supported").set_status(400)
 }
 
-pub fn serde(_e: &serde_json::Error) -> HttpApiProblem {
-    // FIXME: Use error to give more detail to the user
+pub fn serde(e: &serde_json::Error) -> HttpApiProblem {
+    // TODO (fixme): Use error to give more detail to the user
```

Comment:
:x:  This TODO can be removed.

Reply:
Why? What changed?

Reply:
I think you now use the error, which is why you removed the underscore :stuck_out_tongue:

Reply:
ok I guess so, whoever wrote this comment speak now or hold your peace forever.

Reply:
No it is not resolved.

The comment means: return details to the user like `field blublub was unexpected`.

That requires some more work I guess and should be tracked in its own ticket and not in a TODO :)

I'd suggest: remove the TODO and create a proper ticket.

Reply:
#754 opened